packages/menu/src/MenuItem.tsx

```diff
@@ -158,6 +158,11 @@ export interface MenuItemProps {
    */
   children?: React.ReactNode;
 
+  /**
```

Comment:
Okay cool, I figured it might be one or the other, and so decided to air on the side of caution. I will remove this prop entirely. Thanks for clarifying.

Reply:
Resolved in 1bcbfd1